### DISCUSSION TITLE
Remove heat map on mobile for location. On mobile, show location map like in web interface

### DISCUSSION THREAD

**Discussion by sowensupgrade** [immich-app/immich#14934] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Heat map is very annoying and doesn't add anything

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
This has been requested before, we're not planning to change it.
